cookbook/entities/import_export.rst

```diff
@@ -0,0 +1,150 @@
+How to Import and Export Entities
+=================================
+
+You have to create some services and add some configuration to make Oro Platform capable to export
+your custom entities as CSV files and to be able to load data from CSV files for your entities.
+
+All the configuration described below will be added to the ``importexport.yml`` file in the
+``Resources/config`` directory of your application bundle. Make sure that you have a container
+extension class in your bundle that loads the configuration file:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/DependencyInjection/AppExtension.php
+    namespace AppBundle\DependencyInjection;
+
+    use Symfony\Component\Config\FileLocator;
+    use Symfony\Component\DependencyInjection\ContainerBuilder;
+    use Symfony\Component\DependencyInjection\Extension\Extension;
+    use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+    class AppExtension extends Extension
+    {
+        public function load(array $configs, ContainerBuilder $container)
+        {
+            $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/Resources/config'));
+            $loader->load('importexport.yml');
+        }
+    }
+
+Setting Up the Import and Export Processor
+------------------------------------------
+
+Import and export is handled by so called processors which transform the imported data into actual
+entities and vice versa. The easiest way to quickly set up import and export processors for your
+entities is to reuse the :class:`Oro\\Bundle\\ImportExportBundle\\Processor\\ImportProcessor` and
+:class:`Oro\\Bundle\\ImportExportBundle\\Processor\\ExportProcessor` classes that ship with the
+OroImportExportBundle. All you need to do is creating services that are based on abstract services
+from the OroImportExportBundle and let them know which entity class they have to handle:
+
+.. code-block:: yaml
+    :linenos:
+
+    # src/AppBundle/Resources/config/importexport.yml
+    services:
+        app.importexport.processor.export:
+            parent: oro_importexport.processor.export_abstract
+            calls:
+                - [setDataConverter, ['@oro_user.importexport.data_converter']]
```

Comment:
I think it is more correct to use own definition of data convertor than use from user bundle:
    app.importexport.data_converter:
        parent: oro_importexport.data_converter.configurable